src/main/java/sirius/biz/tenants/BizInterceptor.java

```diff
@@ -46,11 +55,23 @@ public boolean beforePermissionError(String permission,
         } else {
             ctx.respondWith()
                .template("/templates/wondergem/error.html.pasta",
-                         NLS.fmtr("BizInterceptor.missingPermission").set("permission", permission).format());
+                         NLS.fmtr("BizInterceptor.missingPermission")
+                            .set("permission", getTranslatedPermission(permission))
+                            .format());
         }
         return true;
     }
 
+    private String getTranslatedPermission(String permission) {
+        if (roles.contains(permission)) {
```

Comment:
couldn't we simply cascade some smartGets? or rather always use the same i18n prefix for all roles?